signal/mod.ts

```diff
@@ -3,6 +3,7 @@
  * Higher level API for dealing with OS signals.
  *
  * @module
+ * @deprecated (will be removed in 1.0.0) Use the Deno signals API instead
```

Comment:
Actually, perhaps it'd be best if we were to link to the OS Signals API documentation here.